api/apps/api/src/modules/projects/project.api.entity.ts

```diff
@@ -154,12 +154,9 @@ export class Project extends TimeUserEntityMetadata {
   @OneToMany((_type) => Scenario, (scenario) => scenario.project)
   scenarios?: Scenario[];
 
-  @ApiPropertyOptional({ type: () => CostSurfaceApiEntity })
-  @OneToOne(
-    (_type) => CostSurfaceApiEntity,
-    (costSurface) => costSurface.project,
-  )
-  costSurface?: CostSurfaceApiEntity;
+  @ApiPropertyOptional({ type: () => CostSurface })
+  @OneToOne((_type) => CostSurface, (costSurface) => costSurface.project)
```

Comment:
please note, this should not be `@OneToOne()` - many cost surfaces can have the same `project_id`, and conversely here this would be `costSurface!: CostSurface[]` (which can very well be an empty array until we get to creating at least the default cost surface for a new project).

Reply:
Yep, both this and above are unfinished (and untested) changes in a rush to wrap this up before the eagle caught me, but it was fast :(